code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1497,13 +1497,19 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				target_pool = istype(target_shove_turf, /turf/open/indestructible/sound/pool) ? target_shove_turf : null
 				shove_blocked = TRUE
 
-		if(target.IsKnockdown() && !target.IsParalyzed())
-			target.Paralyze(SHOVE_CHAIN_PARALYZE)
-			target.visible_message("<span class='danger'>[user.name] kicks [target.name] onto [target.p_their()] side!</span>",
-				"<span class='danger'>[user.name] kicks you onto your side!</span>", null, COMBAT_MESSAGE_RANGE)
-			addtimer(CALLBACK(target, /mob/living/proc/SetKnockdown, 0), SHOVE_CHAIN_PARALYZE)
-			log_combat(user, target, "kicks", "onto their side (paralyzing)")
-
+		if(target.IsKnockdown())
+			var/target_held_item = target.get_active_held_item()
+			if(target_held_item)
+				if(!(target.a_intent == INTENT_GRAB))
+					target.drop_all_held_items()
+					target.visible_message("<span class='danger'>[user.name] kicks \the [target_held_item] out of [target]'s hand!</span>",
+									"<span class='danger'>[user.name] kicks \the [target_held_item] out of your hand!</span>", null, COMBAT_MESSAGE_RANGE)
+					log_combat(user, target, "disarms [target_held_item]")
+					return
+				else
+					target.visible_message("<span class='danger'>[user.name] attempts to kick \the [target_held_item] away, but [target] maintains their grip!</span>",
+									"<span class='danger'>[user.name] fails to kick \the [target_held_item] out of your hand!</span>", null, COMBAT_MESSAGE_RANGE)
+					return
```

Comment:
I don't think these should return and should instead continue, so you can push someone into a bin and make them drop items at the same time

Reply:
👍